src/main/java/uk/ac/cam/cl/dtg/util/ReflectionsUtil.java

```diff
@@ -0,0 +1,53 @@
+package uk.ac.cam.cl.dtg.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ReflectionsUtil {
```

Comment:
Very nitpicky, but this one should be called `ReflectionUtils`, I think. Reflections, in plural, was the name of the library, but we're no longer using that library, so `Reflection` should be singular. `Utils`, on the other hand, should be in plural, because that seems to be the convention.